libraries/AP_HAL_Linux/Util.h

```diff
@@ -48,7 +48,7 @@ class Util : public AP_HAL::Util {
      */
     void set_system_clock(uint64_t time_utc_usec);
     const char* get_custom_log_directory() { return custom_log_directory; }
-    const char* get_custom_terrain_directory() { return custom_terrain_directory; }
+    const char* get_custom_terrain_directory() const final { return custom_terrain_directory; }
```

Comment:
Can we add override please? (Unfortunately) Being final doesn't guarantee it is overriding a virtual method.